### ISSUE TITLE
Replace gulp-util

### ISSUE THREAD

**Issue by fosslinux** [encryptic-team/encryptic#64]
gulp-util is now deprecated, it should be replaced.

> gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

**Comment by Maks-s** [comment]
gulp-util is already removed, see #42

**Comment by fosslinux** [comment]
Hmm, it can't be removed fully, because when I tried to build it I got an error that gulp-util was required.

**Comment by daed** [comment]
It's sneaking back in via gulp-electron.  I'll see about alternatives.

**Comment by daed** [comment]
I created a new branch called electronPackaging in which I rewrote the electron packaging gulpfile so that we could remove gulp-electron (which was bringing in gulp-utils) and also gulp-manifest, which we weren't actually using but still had gulp-utils as a dep.  

My new gulp appears to work more or less properly except when it comes to the final repackaging in zip files.  For some reason the zip is corrupt upon reading.  Also, I think I need to find out how the platformResources are being used.  

I will take a further look at this after sleep and merge this when I get it functional.
